src/networks.js

```diff
@@ -19,6 +19,12 @@ const networks = {
     defaultEthNode: ETH_NODE,
     arbitrator: '0x35e7433141D5f7f2EB7081186f5284dCDD2ccacE',
     disputeManager: '0xc1f1c30878de30fd3ac3db7eacdd33a70c7110bd',
+
+    // TODO: Define if we should get these from subgraph
```

Comment:
Finally decided to keep as is, otherwise we would need to search for the 1hive garden and get the info and i think this is a more general piece of information.